.github/workflows/update-gh-pages.yml

```
@@ -1,16 +1,41 @@
-
-name: Update index.html in GH Pages
+name: Custom GH Pages Post-processing
```

We do not follow title capitalization:

```suggestion
name: Custom GitHub pages post-processing
```